app/controllers/api/stateless/payment_helper.rb

```diff
@@ -27,4 +27,32 @@ def transactions_for_member(member)
       customer(member).transactions.one_off
     end
   end
+
+  module GoCardless
+    module_function
+
+    def customer(member)
+      ::Payment::GoCardless::Customer.find_by!(member_id: member.id)
+    end
+
+    def transactions_for_member(member)
+      customer(member).transactions.one_off
+    end
+
+    def subscriptions_for_member(member)
+      customer(member).subscriptions.order('created_at desc')
+    end
+
+    def subscription_for_member(member:, id:)
+      customer(member).subscriptions.find(id)
+    end
+
+    def payment_methods_for_member(member)
+      customer(member).payment_methods.order('created_at desc')
+    end
+
+    def payment_method_for_member(member:, id:)
+      customer(member).payment_methods.find(id)
+    end
+  end
```

Comment:
Is this going to clash with the extracted helper class you created on your last PR?